extension/index.js

```diff
@@ -1,4 +1,4 @@
-$(() => {
+$().ready(() => {
```

Comment:
not a functional change, but I prefer being clear with how things are being initialized. 

tbh, at first glance i didn't expect it to be jquery.